docs/quickstarts/excel-quickstart-jquery.md

```diff
@@ -8,8 +8,71 @@ ms.localizationpriority: high
+- Node.js (the latest LTS version). Visit the [Node.js site](https://nodejs.org/) to download and install the right version for your operating system. To verify if you've already installed these tools, run the commands `node -v` and `npm -v` in your terminal.
+- Office connected to a Microsoft 365 subscription. You might qualify for a Microsoft 365 E5 developer subscription through the [Microsoft 365 Developer Program](https://developer.microsoft.com/microsoft-365/dev-program), see [FAQ](/office/developer-program/microsoft-365-developer-program-faq#who-qualifies-for-a-microsoft-365-e5-developer-subscription-) for details. Alternatively, you can [sign up for a 1-month free trial](https://www.microsoft.com/microsoft-365/try?rtc=1) or [purchase a Microsoft 365 plan](https://www.microsoft.com/microsoft-365/buy/compare-all-microsoft-365-products).
+
+## Create the add-in project
+
+Click the following button to create an add-in project using the Office Add-ins Development Kit for Visual Studio Code. A page that contains the project decription will open in Visual Studio Code.
+
+<button type="button">[Create an add-in in Visual Studio Code](vscode://msoffice.microsoft-office-add-in-debugger/open-specific-sample?sample-id=Excel-HelloWorld-TaskPane-JS)</button>
+
+<img src="../images/office-add-ins-development-kit-samplepage-excelquickstart.png" alt="Screenshot shows the sample page in Visual Studio Code"/>
+
+In the prompted page, select **Create** to create the add-in project. In the **Workspace folder** dialog that opens, select the folder where you want to create the project. 
+
+Office Add-ins Development Kit will create the project. It will then open the project *in a second Visual Studio Code window*. Close the original Visual Studio Code window.
+
+> [!NOTE]
+> You will receive prompts to install the Office Add-ins Development Kit extension if you haven't. You can also [install the Office Add-ins Development Kit manually](https://learn.microsoft.com/en-us/office/dev/add-ins/develop/development-kit-overview?tabs=vscode).
+
+## Explore the project
+
+The add-in project that you've created with the Office Add-ins Development Kit contains sample code for a basic task pane add-in. If you'd like to explore the components of your add-in project, open the project in your code editor and review the files listed below. When you're ready to try out your add-in, proceed to the next section.
+
+1. The ./manifest.xml or manifest.json file in the root directory of the project defines the settings and capabilities of the add-in.
+1. The ./src/taskpane/taskpane.html file contains the HTML markup for the task pane.
+1. The ./src/taskpane/taskpane.css file contains the CSS that's applied to content in the task pane.
+1. The ./src/taskpane/taskpane.js file contains the Office JavaScript API code that facilitates interaction between the task pane and the Office client application.
+
+## Try it out
```

Comment:
"Try it out", "Stop testing", "Troubleshooting" are same as the ones in (docs/develop/development-kit-overview.md)

Reply:
Same suggestion for an include file. This prevents content from being different in two places if we miss something in an update.

Reply:
Is there a reason we didn't include the headers in the include files? I think you could make this whole thing one include file.

Reply:
For the "start debugging" one, there is other content after the header in the dev kit overview doc, so the header is not in the include file. Others are included.